salt/modules/virt.py

```diff
@@ -250,7 +250,7 @@ def _gen_xml(name,
         # TODO: make bus and model parameterized, this works for 64-bit Linux
         context['controller_model'] = 'lsilogic'
 
-    if kwargs.get('enable_vnc', True):
+    if 'enable_vnc' in kwargs:
```

Comment:
@isbm thanks for the review! I tried your approach, but it doesn't work. It never adds the vnc part to the vm xml, doesn't matter if you set enable_vnc or enable_vnc=True. I agree enable_vnc=False yielding true is contradictory, but there is no good reason why a user will need to explicitly set enable_vnc=False (since its the default already).